detection/impact/evenly-timestomped.sql

```diff
@@ -51,5 +51,4 @@ WHERE
   AND file.path NOT LIKE '%/lynis%'
   AND file.path NOT LIKE '%/yelp-xsl%'
   AND file.path NOT LIKE '/etc/cups/%'
-  AND file.path NOT LIKE '/usr/share/libinput/%.quirks'
-  AND file.path NOT LIKE '/usr/lib64/electron/locales/%.pak'
+  AND file.path NOT LIKE '/usr/%'
```

Comment:
Instead of filtering it out here, exclude it at line 29 so that the directory isn't scanned at all.